### PR TITLE
[cocoon] Exclude null builds when calling buildbucket API

### PR DESCRIPTION
**Pull request by keyonghan**
When buildbucket API call returns empty builds, it fails `/api/refresh-chromebot-status` which refresh LUCI builds statuses to flutter dashboard.
This is to exclude the empty case.

https://github.com/flutter/flutter/issues/79380

### CODE REVIEW

File: app_dart/lib/src/service/luci.dart
Comment (godofredoc) [inline app_dart/lib/src/service/luci.dart:205]: This will only hide the problem. Do we know why some builds are returning null?

**Comment by keyonghan** [comment]
This is to open the tree first.
A following up is to find the reason why. Otherwise that empty build will never be updated.

File: app_dart/lib/src/service/luci.dart
Comment (keyonghan) [reply app_dart/lib/src/service/luci.dart:205]: This is to open the tree first.
A following up is to find the reason why. Otherwise that empty build will never be updated.

Comment (godofredoc) [reply app_dart/lib/src/service/luci.dart:205]: LGTM if we will be able to find the issue after we land this change

**Review by godofredoc — APPROVED** [review]
LGTM if we will be able to root cause the real issue after this mitigation.

File: app_dart/lib/src/service/luci.dart
Comment (keyonghan) [reply app_dart/lib/src/service/luci.dart:205]: Created https://github.com/flutter/flutter/issues/79393 to track.

**Comment by keyonghan** [comment]
It turns out the builder was enabled in prod_builders.json, but missed from builder config https://devops-console-oss.corp.google.com/flutter/infra/+/master:config/framework_config.star

**Comment by godofredoc** [comment]
> It turns out the builder was enabled in prod_builders.json, but missed from builder config https://devops-console-oss.corp.google.com/flutter/infra/+/master:config/framework_config.star

How can we prevent this from happening again?

**Pull request merged**
